src/main/java/com/netflix/imflibrary/RESTfulInterfaces/IMPValidator.java

```diff
@@ -913,4 +920,76 @@ else if(errorObject.getErrorLevel() == IMFErrorLogger.IMFErrors.ErrorLevels.WARN
         }
         return imfErrorLogger.getErrors();
     }
+
```

Comment:
Add a Javadoc comment for public methods, explaining what it is used for, and what parameters are required.

Reply:
Added.